Regular-Question-Answers/most-visited-sector-in-a-circular-track.cpp

```diff
@@ -0,0 +1,65 @@
+// Author: Murat Biberoğlu
+// Question Link: https://leetcode.com/problems/most-visited-sector-in-a-circular-track/
+
```

Comment:
n<=100 ve m<=100 olduğundan herhalde her döngüyü simüle etmek fazla zaman almıyor ama zaten circular olduğundan ve tur sayısını başlı başına önemi olmadığından yalnızca baştaki ve sondaki durumu gözetip, arada oluşan tam daireleri es geçebilirsin, böylece sort result count vektörleri yerine yalnız bir result yeterli olur. Space comp da bariz iyileşme olur.
Ayrıca space ve time complexity leri eklemelisin.

Reply:
evet haklısın aslında m ve n çok küçük olduğundan bu O(10^4) te çalışıyor ve counts vectoründe de en fazla 100 integer var dolayısıyla maximum 400 bytelık bir memory var ancak bu sınırlar için optimize olduğundan değiştirmeye gerek var mı bilemedim.

Reply:
tamam öyleyse sen bilirsin :) zaten hint'ler de dediğine yönelik imiş

Reply:
soru giriş seviyesinde sonuçta ama m ve n büyük olsaydı dediğini kesinlikle yapmalıydık. time ve memory complexitylerini ekleyeceğim bu arada.